src/monitor.py

```diff
@@ -61,14 +61,18 @@ def _run(self, sub_id):
             event = self._api.receive_event(sub_id)
             obj = event.data
             dt = datetime.datetime.fromisoformat(event['time'])
-            print(self.LOG_FMT.format(
-                time=dt.strftime('%Y-%m-%d %H:%M:%S.%f'),
-                commit=obj['revision']['commit'][:12],
-                id=obj['id'],
-                state=state_map[obj['state']],
-                result=result_map[obj['result']],
-                name=obj['name']
-            ), flush=True)
+            # Print the status line only for events that contain
+            # "revision" data. Implement proper event filtering when the
+            # models are production-ready
+            if 'revision' in obj:
```

Comment:
Same comment as https://github.com/kernelci/kernelci-core/pull/2212.
Events on `node` channel will always contain `revision` field.
What are the non-node events?

Reply:
Sure. It would be nice if you could create a Github issue for this to keep track.